ssm/helpers.go

```diff
@@ -18,23 +18,19 @@ import (
 )
 
 //CreateSSMDescribeInstanceInput returns an *ssm.DescribeInstanceInformationInput object for use when calling the DescribeInstanceInformation() method
-func CreateSSMDescribeInstanceInput(filters []map[string]string, instances CommaSlice) *ssm.DescribeInstanceInformationInput {
-	var ssmInputFilters []*ssm.InstanceInformationStringFilter
-
-	if len(filters) > 0 {
-		for _, v := range filters {
-			//if you have multiple filter groups, this drops all but the last
-			BuildFilters(v, &ssmInputFilters)
-			// Build our filters based on --filter and --instances flags
-		}
-	}
+func CreateSSMDescribeInstanceInput(filters map[string]string, instances CommaSlice) *ssm.DescribeInstanceInformationInput {
+	var iisFilters []*ssm.InstanceInformationStringFilter
+
+	//if you have multiple filter groups, this drops all but the last
+	BuildFilters(filters, &iisFilters)
```

Comment:
BuildFilters is never used outside of the ssm package. I suggest not exporting that function since it is used here and within a test within the same package.